docs/functions.rst

```diff
@@ -1758,6 +1758,64 @@ For this phase, we will use the ``backupIdentifier`` Artifact provided by backup
       backupID: "{{ .ArtifactsIn.backupIdentifier.KeyValue.id }}"
       image: ghcr.io/kanisterio/kanister-tools:0.89.0
 
+.. _backupdataallusingkopiaserver:
+
+BackupDataAllUsingKopiaServer
+-----------------------------
+
+This function concurrently backs up data from one or more pods into an any
+object store supported by Kanister using Kopia Repository Server as a data mover.
+
+.. note::
+   It is important that the application includes a ``kanister-tools``
+   sidecar container. This sidecar is necessary to run the
+   tools that capture path on a volume and store it on the object store.
+
+   Additionally, in order to use this function, a RepositoryServer CR is needed
+   while creating the :ref:`actionsets`
+
```

Comment:
Let's also mention the document that, if we have any, on how to create the repo server CR.